iterative-walkthrough/deployment-1/main.tf

```diff
@@ -9,7 +9,7 @@ resource "layer0_environment" "demo" {
 }
 
 module "guestbook" {
-  source         = "github.com/quintilesims/layer0-examples//guestbook/module"
+  source         = "github.com/quintilesims/layer0-examples/guestbook/module"
```

Comment:
Terraform has you place a double slash `//` if the terraform module is in a subdir of a repo, otherwise it clones the entire  repo.